pkg/kubelet/apis/config/scheme/testdata/KubeletConfiguration/roundtrip/default/v1beta1.yaml

```diff
@@ -86,6 +86,7 @@ seccompDefault: false
 serializeImagePulls: true
 shutdownGracePeriod: 0s
 shutdownGracePeriodCriticalPods: 0s
+singleProcessOOMKill: false
```

Comment:
drop this field, we want to make sure absence of the field is round-tripped / preserved by API-level defaulting